nav2_bringup/bringup/params/nav2_params_selectable_components_test.yaml

```diff
@@ -0,0 +1,317 @@
+amcl:
```

Comment:
I don't think you meant to commit this file? I think you could add the 2 to the main config file (though named better!) 

Either way, the config files in this directory will need an update for the change in type of the goal checker parameter to a vector

Reply:
Indeed, I did not mean to add anything not related with our changes. I will fix that.
Just tell me a better name for this file (I am not very good on naming).

Correct, main files also will need changes to support a vector for goal checkers. That is pending.

Reply:
I updated the yaml files taking into account your notes.

Besides, the debugging of the solution is still pending but it is in progress.